.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: ['eslint:recommended', 'prettier'],
+  ignorePatterns: ['dist'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    allowImportExportEverywhere: true,
```

Comment:
Curious what `allowImportExportEverywhere` does 😅

Reply:
I'm actually not 100% sure, removing it doesnt seem to cause an issue though

This was the documentation I found, might be something outdated we can remove - https://babeljs.io/docs/babel-parser

Reply:
If we don't have a use case for it then think we may as well remove. Ideally would like to get the UI package to use the shared linting config in the future since the existing one is kind of hacky